test/bench/bench.cpp

```diff
@@ -110,9 +108,9 @@ struct benchmark_case
 };
 
 
-void load_benchmark(fs::path path)
+void load_benchmark(fs::path path, const fs::path& subdir)
 {
-    auto base_name = path.stem().string();
+    const auto base_name = (subdir / path.stem()).string();
```

Comment:
Why `subdir` parameter is now needed? can't full path be passed in `path` only?

Reply:
I want the tests to be named `subdir/filename/...`. I could also pass the root dir but this way seems easier.

Reply:
I think I got it, you want the test names not to have the full path, but relative one. But it's confusing. Maybe this parameter shouldn't be `fs::path` at all (but string) if it's used only for test name construction? (could still be `fs::path` in `load_benchmarks_from_dir`)
Maybe I'd call it `test_name_prefix` and move `(subdir / path.stem()).string()` part to the calling side.

Reply:
Make sense.

Reply:
Done.